app/views/users/edit.html.erb

```diff
@@ -0,0 +1,10 @@
+ <%= form_for(@user, as: :user, url: user_path, authenticity_token: form_authenticity_token)do |user|%>
+   <%= render 'shared/error_messages' %>
+  <%= user.label(:username, 'Username') %>
```

Comment:
The same also applies here we don't need authenticity token or as or url arguments.